src/main/java/fi/aalto/cs/apluscourses/ui/base/SingleSelectionList.java

```diff
@@ -14,10 +17,13 @@
   public final transient Bindable<SingleSelectionList<T>, T[]> listDataBindable =
       new Bindable<>(this, SingleSelectionList::setListData);
 
+  private final transient Runnable submitRunnable;
+
   /**
    * Constructs a new SingleSelectionListener.
```

Comment:
Nit: Probably my mistake originally, but: "listener"?

Reply:
Do you mean I should call the field "listener"?

Reply:
Nah, just a wrong name for the class in the javadoc :P